### PR TITLE
[Network Config Mgr] Add NodeInfo APIs

### PR DESCRIPTION
**Pull request by pkommoju**
NodeInfo cache in NCM is working.
Bulk API is not tested.

### CODE REVIEW

File: services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.futurewei.alcor.netwconfigmanager.util;
+
+public class NetworkConfigManagerConstants {
```
Comment (xieus) [inline services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java:3]: Regarding the naming, I would recommend to make it short, for example, Constants. 

The full class name include NetworkConfigManager in the path.

File: scripts/ncmtest.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
```
Comment (xieus) [inline scripts/ncmtest.sh:2]: Could we create a new folder /scripts/test and move this script there?

**Review by xieus — APPROVED** [review]
LGTM with two minor comments.

File: services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java (hunk 1 above)
Comment (pkommoju) [reply services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java:3]: You mean, rename the class to Constants? I think that makes sense. I will address in the next commit.

File: scripts/ncmtest.sh (hunk 2 above)
Comment (pkommoju) [reply scripts/ncmtest.sh:2]: Makes sense. I will do that.

File: services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java (hunk 1 above)
Comment (xieus) [reply services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java:3]: Right.

File: scripts/ncmtest.sh (hunk 2 above)
Comment (pkommoju) [reply scripts/ncmtest.sh:2]: Are you waiting me to fix these in this branch/PR or can I do it after you merge into master?

**Comment by xieus** [comment]
This PR has some Github merge issue. A new PR #572 is created.

**Pull request closed without merging**